docs/getting-started-guide.org

```diff
@@ -265,6 +265,81 @@ To list all the available configuration modules, you can run:
 ls ./modules/*-config.el
 #+end_src
 
+*** Disabling Crafted Emacs Modules
+
+If you no longer want to use a module provided by Crafted Emacs anymore, you can
+ disable it by completing the following two steps
+
+1. Stop loading the module in your Emacs initialization file
+2. Uninstall the packages installed by the module
+
+For example, if you previously enabled the =crafted-completions-packages=
+and =crafted-completion-config= modules, find and remove the corresponding
+=require= expressions from your Emacs initialization file:
+
+#+begin_src emacs-lisp
+(require 'crafted-completion-packages)
+
+(require 'crafted-completion-config)
+#+end_src
+
+Next, open the =crafted-completion-packages.el= file and note
+all packages specified in
+=(add-to-list 'package-selected-package ...)= expressions.
+
+Finally, uninstall these packages using either:
+
+- The command ~M-x package-delete~
+- Or the =pakage.el= UI (to open the UI type ~M-x list-packages~)
+
+**** Disabling Only Specific Packages from a Crafted Emacs Module
```

Comment:
This section suggests putting things into the `init.el` file, which is fine, but possibly an "easier" approach is to copy the crafted module to the `custom-modules/` folder and then edit the file from there.  Modules found in the users `custom-modules/` folder are loaded before the ones found in `crafted-emacs`. Thus the `init.el` file does not have to change.  Perhaps we need a generic description of how to customize Crafted Emacs using a method like this and then we can say "see <that chapter>" with some specific commentary on this specific task. 

Does that makes sense?

Reply:
Correct, the folder is added to the `load-path` only.  The user must load the modules in `custom-modules` with a `require` form (or other ways, but this is the most common way).  

What I mean is this, 

```emacs-lisp
(require 'crafted-ide-packages)
```

does not have to change.  You can copy the `crafted-ide-packages.el` file to your own `custom-modules/` folder, edit there.  Crafted Emacs will find and load the `crafted-ide-packages` from the `custom-modules/` folder as it is earlier in the `load-path`.  Does that help?

Reply:
I've updated the section. Please take a look.